website/blog/authors.yml

```diff
@@ -19,8 +19,11 @@ slorber:
   title: Docusaurus maintainer, This Week In React editor
   url: https://thisweekinreact.com
   image_url: https://github.com/slorber.png
-  twitter: sebastienlorber
   email: sebastien@thisweekinreact.com
+  socials:
```

Comment:
Other users have a Twitter attribute too

Maybe we should name it "x" right now considering it's unlikely they change back their name now